desktop/src/edu/cornell/gdiac/optimize/desktop/DesktopLauncher.java

```diff
@@ -7,9 +7,9 @@
 public class DesktopLauncher {
 	public static void main (String[] arg) {
 		GDXAppSettings config = new GDXAppSettings();
-		config.title = "Optimization";
-		config.width  = 800;
-		config.height = 600;
+		config.title = "TempoRary";
+		config.width  = 1200;
+		config.height = 800;
```

Comment:
I'm assuming this screen size is so that it's visible on your computer. We need to make a note to change the graphic.